apps/builder/app/builder/features/style-panel/sections/transitions/transition-utils.ts

```diff
@@ -86,6 +89,21 @@ export const findTimingFunctionFromValue = (
   );
 };
 
+export const getAnimatablePropertiesOnInstance = (
+  currentStyle: StyleInfo
+): Set<string> => {
+  const properties: Set<string> = new Set();
+  let property: StyleProperty;
+  for (property in currentStyle) {
+    const value = currentStyle[property];
+    const prop = hyphenateProperty(property);
+    if (isAnimatableProperty(prop) === true && value?.local !== undefined) {
+      properties.add(prop);
+    }
```

Comment:
Right now, it is doing a very basic check if the values are set in local to the node. And the currentStyle style info seems to respect the breakpoints too when changed.

Opened the PR to give a brief experience of the UX for this. Please let me know the other cases that we can improve here.